labgrid/driver/rawnetworkinterfacedriver.py

```diff
@@ -0,0 +1,96 @@
+# pylint: disable=no-member
+import json
+import os
+import signal
+import subprocess
+
+import attr
+
+from .common import Driver
+from ..factory import target_factory
+from ..step import step
+from ..util import sshmanager
+from ..util.helper import processwrapper
+from ..util.managedfile import ManagedFile
+from ..resource.remote import RemoteNetworkInterface
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class RawNetworkInterfaceDriver(Driver):
+    bindings = {
+        "iface": {"NetworkInterface", "RemoteNetworkInterface", "USBNetworkInterface"},
+    }
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self.proc_pcap_map = {}
+
+    def _get_wrapper_prefix(self):
+        return self.iface.command_prefix + ["sudo", "labgrid-raw-interface"]
+
+    def _remote_tmp_file_name(self):
+        cmd = self.iface.command_prefix + ["mktemp", "-d"]
+        return processwrapper.check_output(cmd).strip().decode("utf-8") + "remote.pcap"
```

Comment:
On my system, `mktemp -d` does not produce a trailing slash, so we need to use `os.path.join()` instead of string concatenation.

Reply:
Code was dropped with the change to capturing to stdout, see above.